### ISSUE TITLE
MWE not working

### ISSUE THREAD

**Issue by MaAl13** [dietmarwo/fast-cma-es#24]
Helo, i am trying to run your optimization on a very simple Rosenbrock function problem, but it fails.Can you tell me what is wrong with my code?

```
import numpy as np
from scipy.optimize import Bounds
from fcmaes.optimizer import De_cpp
from fcmaes import retry


def test_fun(params):
    return np.sum(100.0 * ((params[1:]+1) - (params[:-1]+1)**2.0)**2.0 + (1 - (params[:-1]+1))**2.0)

# Main optimization function
def optimize_model():
    lb = np.ones(10) * -5  # Lower bound set to -5
    ub = np.ones(10) * 10  # Upper bound set to 10
    bounds = Bounds(lb, ub)
    result = retry.minimize(test_fun, bounds, num_retries=1, optimizer=De_cpp(1000))
    return result

# Run the optimization
if __name__ == "__main__":
    result = optimize_model()
    print(result)
    print(result.x)
    print(result.fun)
```

Returns:
'NoneType' object is not subscriptable
2024-07-31 14:14:28.382 | DEBUG    | fcmaes.retry:dump:388 - 0.07 0 1 0 179769313486231570814527423731704356798070567525844996598917476803157260780028538760589558632766878171540458953514382464234321326889464182768467546703537516986049910576551282076245490090389328944075868508455133942304583236903222948165808559332123348274797826204144723168738177180919299881250404026184124858368.000000 0.00 0.00 [] [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
 success: True
     fun: 1.7976931348623157e+308
       x: [ 0.000e+00  0.000e+00  0.000e+00  0.000e+00  0.000e+00
            0.000e+00  0.000e+00  0.000e+00  0.000e+00  0.000e+00]
    nfev: 0
[0. 0. 0. 0. 0. 0. 0. 0. 0. 0.]
1.7976931348623157e+308

**Comment by ymLuo1214** [comment]
Meet the same problem, have you fixed that?

**Comment by MaAl13** [comment]
Nope, i switched to another package, which works really well on my problem. It is called evosax if you want to try it out.

**Comment by dietmarwo** [comment]
checked on Linux using 1.6.11 and got the result below:
Did you try "pip install fcmaes --upgrade" ?
Do you use Windows?

2024-11-12 14:20:26.878 | DEBUG    | fcmaes.retry:dump:400 - 0.07 14771 1 1034 6.584272 0.00 0.00 [] [-1.9525107036864022, -0.03372250053045583, -0.055189729344992514, -0.1215086779748337, -0.22202042973813602, -0.39903601766528296, -0.6439347055511802, -0.862776492275128, -0.9657059723528973, -0.9817403814273438]
2024-11-12 14:20:26.886 | DEBUG    | fcmaes.retry:add_result:342 - 0.08 12925 1 1034 6.584272 6.584272 6.58 0.00
2024-11-12 14:20:26.933 | DEBUG    | fcmaes.retry:dump:400 - 0.13 7953 1 1034 6.584272 6.58 0.00 [0.0] [-1.9525107036864022, -0.03372250053045583, -0.055189729344992514, -0.1215086779748337, -0.22202042973813602, -0.39903601766528296, -0.6439347055511802, -0.862776492275128, -0.9657059723528973, -0.9817403814273438]
 success: True
     fun: 6.584271797344801
       x: [-1.953e+00 -3.372e-02 -5.519e-02 -1.215e-01 -2.220e-01
           -3.990e-01 -6.439e-01 -8.628e-01 -9.657e-01 -9.817e-01]
    nfev: 1034
[-1.9525107  -0.0337225  -0.05518973 -0.12150868 -0.22202043 -0.39903602
 -0.64393471 -0.86277649 -0.96570597 -0.98174038]
6.584271797344801

**Comment by dietmarwo** [comment]
By the way, you can get rid of the debug messages using something like

```
logger.remove()
logger.add(sys.stdout, format="{time:HH:mm:ss.SS} | {process} | {level} | {message}", level="INFO")
logger.add("log_{time}.txt", format="{time:HH:mm:ss.SS} | {process} | {level} | {message}", level="INFO")
```
